index.js

```diff
@@ -17,349 +21,408 @@ var cluster = new couchbase.Cluster(
     username: 'Administrator',
     password: 'password'
   }
-);
+)
 
 // Open a specific Couchbase bucket, `travel-sample` in this case.
-var bucket = cluster.bucket('travel-sample');
-// And select the default collection
-var coll = bucket.defaultCollection();
+var bucket = cluster.bucket('travel-sample')
 
 // Set up our express application
-var app = express();
-app.use(cors());
-app.use(bodyParser.json());
+var app = express()
+app.use(morgan('dev'))
+app.use(cors())
+app.use(express.json())
+
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 
-// Serve the public directory to the root of the web server.
-app.use(express.static('public'));
+var tenants = express.Router({mergeParams: true})
 
 function authUser(req, res, next) {
   bearerToken()(req, res, () => {
     // Temporary Hack to extract the token from the request
-    req.token = req.headers.authentication.split(' ')[1];
+    req.token = req.headers.authorization.split(' ')[1]
     jwt.verify(req.token, JWT_KEY, (err, decoded) => {
       if (err) {
-        res.status(400).send({
+        return res.status(400).send({
           error: 'Invalid JWT token',
```

Comment:
In the swagger api docs I have the error model as `{"message": "error message x"}`, is this what will be returned by the Node api? 

Or will it return something like this?
```
{"error": "Invalid JWT token", cause: "some cause"}
```

Reply:
the frontend seems to only read `error` and `context` fields in the case of an error, does the python app act differently?

I'll go through the swagger tests and check the example results, haven't done that yet!